src/components/menu/Menu.js

```diff
@@ -0,0 +1,97 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import saveItem from '../../api/saveItem';
+import _cloneDeep from 'lodash/cloneDeep';
+import getUpdatedQuantityPerDishById from './helpers/getUpdatedQuantityPerDishById';
+import getUpdatedCartItems from './helpers/getUpdatedCartItems';
+
+//Constants
+import { INITIAL_STATE } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = _cloneDeep(INITIAL_STATE);
+  resetCartItems = () => {
+    this.setState({
+      ...INITIAL_STATE,
```

Comment:
don't do this, explicitly mention state

1. it's not readable
2. tomorrow if someone adds another key into INITIAL_STATE which may or may not be required to set here this code will break